### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by jessesquires**

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -108,10 +108,10 @@ GEM
     unicode-display_width (1.7.0)
 
 PLATFORMS
-  ruby
+  x86_64-darwin-19
```
Comment (anaulin) [inline Gemfile.lock:111]: Is this change intentional? I am not familiar with the effects / consequences of changing the `PLATFORMS` section in the `Gemfile.lock`, so maybe this is ok, just checking. 😸

**Review by anaulin — APPROVED** [review]
Happy to have this merged if you are confident it works (and the CI seems to pass), but see below for my question -- I don't know what are the consequences of removing `ruby` from the `PLATFORMS` section, and I'm wondering if that's intentional.

**Comment by jessesquires** [comment]
Yeah, it's a new thing in Bundler 2.2: https://bundler.io/blog/2020/12/09/bundler-v2-2.html

> Bundler now needs to record the specific platform it used to resolve a given Gemfile in the lockfile, so you will notice some source control differences there when you upgrade. Commit these changes for all your supported platforms to make sure all developers of your application get a consistent resolution under all platforms.

No ill-effects. Although, if someone runs `bundle install` from a different platform, it create a diff where that new platform gets appended to this list. (which won't effect anything)

Some gems offer platform-specific versions.
